cmd/airflow.go

```diff
@@ -463,12 +466,21 @@ func newObjectExportCmd() *cobra.Command {
 
 // Use project name for image name
 func airflowInit(cmd *cobra.Command, args []string) error {
+	// The first positional argument is the project name.
+	// If the project name is provided in this way, we'll
+	// attempt to create a directory with that name.
+	if len(args) > 0 {
+		projectName = args[0]
+	}
```

Comment:
This might mean that if someone is trying `astro dev init --name test project`, would result in a project name as `project` when they might be looking for `test project`, should we at least warn and confirm if not error, when someone is setting both the flag and arg?

Reply:
Good catch. Updated the logic here to catch this case and throw an error.

Reply:
Also added a corresponding test.